definitions/divorce/json/CaseEventToFields.json

```diff
@@ -6867,5 +6867,70 @@
     "PageLabel": "Create a Notice of Acting",
     "PageDisplayOrder": 1,
     "ShowSummaryChangeOption": "Yes"
+  },
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "GrantDecreeAbsolute",
+    "CaseFieldID": "D8PetitionerFirstName",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": "grantDA",
+    "PageLabel": "Grant Decree Absolute",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "GrantDA=\"potato\""
```

Comment:
As much as this has made me smile, I think "NeverShow" is clearer for new joiners to understand.